var/spack/repos/builtin/packages/py-continuum/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyContinuum(PythonPackage):
+    """A clean and simple data loading library for Continual Learning"""
+
+    homepage = "https://continuum.readthedocs.io/en/latest/"
+    pypi = "continuum/continuum-1.2.7.tar.gz"
+
+    maintainers = ["thomas-bouvier"]
+
+    version("1.2.7", sha256="baadcdbe2f5b3c05254307c73434a012f2a3d663e1be9469a03d4b82559e98e1")
+
+    depends_on("python@3.6:")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-torch@1.2.0:", type=("build", "run"))
+    depends_on("py-torchvision@0.4.0:", type=("build", "run"))
+    depends_on("py-numpy@1.17.2:", type=("build", "run"))
+    depends_on("pil@6.2.1:", type=("build", "run"))
+    depends_on("py-matplotlib@3.1.0:", type=("build", "run"))
+    depends_on("py-scipy@1.3.3:", type=("build", "run"))
+    depends_on("py-scikit-image@0.15.0:", type=("build", "run"))
+    depends_on("py-scikit-learn@0.24.1:", type=("build", "run"))
+    depends_on("py-pandas@1.1.5:", type=("build", "run"))
+    depends_on("py-pytest@5.0.1:", type=("build", "run"))
+    depends_on("py-pytest-mock@3.6.1:", type=("build", "run"))
```

Comment:
Missing dep:
```python
depends_on("py-prospector@1.2:+with_mypy", type=("build", "run"))
```

Reply:
`prospector` is not called from any Python code in continuum (it's only called as a Makefile target). I opened a PR which has been merged here https://github.com/Continvvm/continuum/pull/280. Do you think it's worth adding this dep in Spack?

Reply:
The only problem with leaving it out is that `pip check` will complain that it's missing. Not really a big deal. Could also add your PR as a patch to the file. I'll leave the final decision to you, I'm fine with whatever since I don't personally use this package.